source/precalculus/source/04-PR/01.ptx

```diff
@@ -200,6 +201,87 @@
     </statement>
   </definition>
 
+  <activity xml:id="standard-to-vertex-form">
+
+  <introduction>
+    <p>
+      Completing the square (see <xref ref="EQ5"/>) is a very useful tool or method to convert the quadratic equation in standard form into vertex form. Let's start with the standard form of <m>y=x^2-4x+3</m> and convert it into vertex form.
+    </p>
+  </introduction>
+
+  <task>
+    <statement>
+      <p>
+        Before we begin, isolate the terms with <m>x</m> on one side of the equation. What equation do you now have?
+      </p>
+    </statement>
+    <answer>
+      <p>
+        <m>y-3=x^2-4x</m>
+      </p>
+    </answer>
+  </task>
+
+  <task>
+    <statement>
+      <p>
+        Notice that because <m>a=1</m>, apply the "completing the square" steps to determine the constant term that is added to both sides of the equation.
+      </p>
+    </statement>
+    <hint>
+      <p>
+        Refer back to <xref ref="def-completing-the-square"/> to help you determine how to find the constant term.
+      </p>
+    </hint>
+    <answer>
+      <p>
+        By using the "completing the square" steps, students should get:
+        <me>
+          y-3+(4)=x^2-4x+(4)
+        </me>
+        <me>
+          y+1=x^2-4x+4
+        </me>
+      </p>
+    </answer>
+  </task>
+
+  <task>
+    <statement>
+      <p>
+        Factor the perfect square trinomial. 
+      </p>
+    </statement>
+    <answer>
+      <p>
+        <me>
+          y+1=(x-2)^2
+        </me>
+      </p>
+    </answer>
+  </task>
+
+  <task>
+    <statement>
+      <p>
+        Now isolate <m>y</m> and then determine the vertex and axis of symmetry.
+      </p>
+    </statement>
+    <answer>
+      <p>
+        <m>y=(x-2)^2-1</m>
+      </p>
+      <p>
+        Axis of Symmetry: <m>x=2</m>
+      </p>
+      <p>
+        Vertex: <m>(2,-1)</m>
+      </p>
+    </answer>
+  </task>
+    
+  </activity>
+
```

Comment:
See comment in #381!

Reply:
Pasting it here so it's easier:

> I added an activity to have students convert from standard to vertex. Does that work?

I mentioned this mostly to build the idea in general. To develop the fact that the x value of the vertex is -b/2a by completing the square on y=ax^2+bx+c, not a specific quadratic. Currently, the text just says x=-b/2a without any reasoning behind it, so I thought completing the square to get to vertex form would show that -b/2a pop up where we expect the h in vertex form.

As is, it seems like this activity is encouraging students that the way to find the vertex and aos of a standard form quadratic is by completing the square, which is not we'd actually want students to do.

I mentioned this could be in an appendix because maybe it's too in the weeds for a lot of people since there is a lot of algebraic manipulation of the a, b, and c. Or if it's in the text, maybe show the whole thing, and then ask what the x-value of the vertex is?

Reply:
Oh ok, I misinterpreted that then. I vote that if we do include it (with the general quadratic), we put it in the appendix. When teaching this course, I don't normally have students work on the general case.

Reply:
@kathypinzon @siwelwerd 

Do you guys have a preference? I also think it's nice to have students do a specific example to show that the two forms are related. While we don't normally want students to find the vertex and such in this way, it's nice to see how to move from one form to another.

Reply:
>As is, it seems like this activity is encouraging students that the way to find the vertex and aos of a standard form quadratic is by completing the square, which is not we'd actually want students to do.

I think if you asked me to find the vertex and axis of symmetry of a quadratic, I'd complete the square... what are you wanting your students to do?

Reply:
I think most people would either have students find the vertex and AOS from vertex form or tell them to graph it and find it on the graph.

Reply:
But how do you get vertex form?  Isn't that the completing the square part?

Reply:
> > As is, it seems like this activity is encouraging students that the way to find the vertex and aos of a standard form quadratic is by completing the square, which is not we'd actually want students to do.
> 
> I think if you asked me to find the vertex and axis of symmetry of a quadratic, I'd complete the square... what are you wanting your students to do?

I would use x=-b/2a over completing the square pretty much every time. And that's what I show my students too as a method if they had standard form. The idea I had in the suggestion was to show where -b/2a comes from (which is completing the square on the standard form to get to vertex form). I would not expect a student to complete the square to find the vertex, though it would be fine with me if they did.

Reply:
> I think most people would either have students find the vertex and AOS from vertex form or tell them to graph it and find it on the graph.

I would tell them to use -b/2a! I do show them the completing the square bit, but I don't have them do it because it's messy and frustrating and not something I'd ask them to do again in a general sense.

Reply:
> @kathypinzon @siwelwerd
> 
> Do you guys have a preference? I also think it's nice to have students do a specific example to show that the two forms are related. While we don't normally want students to find the vertex and such in this way, it's nice to see how to move from one form to another.

I'm fine with keeping the specific example in there, but I think it needs an additional question at the end that connects it to -b/2a to motivate the observation that the vertex is (-b/2a,f(-b/2a)). And I think it would be easier to see that with an a other than 1. Otherwise is just looks like the vertex is -b/2.